pkg/kvstore/kvstore.go

```diff
@@ -88,8 +88,10 @@ func Setup(dbPort int, serverPort int, clearKeys bool) {
 	grpcServer := grpc.NewServer(opts...)
 	storepb.RegisterKVStoreServer(grpcServer, NewKVStoreServer(client))
 	fmt.Printf("Serving KV Store at localhost:%d", serverPort)
-	err = grpcServer.Serve(lis)
-	if err != nil {
-		fmt.Println(err.Error())
-	}
+	go func(){
```

Comment:
Why do we move this into go routine?

Reply:
In internal/cli/startup/startup.go : its already within a go routine
```
	go func() {
		tagservice.Setup(6379, e.tagPort, e.clearKeys)
	}()
```

Reply:
@seankimkdy As discussed on Discord, `tagservice` now runs on separate go routine. What's your opinion on the matter?